cost/azure/schedule_instance/azure_schedule_instance.pt

```diff
@@ -0,0 +1,537 @@
+         delete all_tags.schedule;
+      }
+      if((now > next_start || now > next_stop || next_start==null || next_stop==null) && (schedule !== "")){
+        results.push({
+          "id":instance.id,
+          "state": instance.status,
+          "location": instance.location,
+          "schedule":schedule,
+          "rg":instance.rg,
+          "name":instance.name,
+          "subscriptionId":instance.subscriptionId,
+          "subscriptionName":instance.subscriptionName,
+          "all_tags": resource_tags,
+          "tags":all_tags,
+          "next_start":next_start_iso,
+          "next_stop":next_stop_iso	
+        })
+      }
+    }
+  })
+ EOF
+end
+
+###############################################################################
+# Escalations
+###############################################################################
+
+escalation "esc_email" do
+  automatic true
+  label "Sends Email"
+  description "Sends Incident Email"
+  email $param_email
+end
+
+escalation "esc_schedule_instance" do
+  automatic contains($param_automatic_action, "Schedule Instances")
+  label "Schedule"
+  description "Stop or Start the Instance"
+  run "schedule_instance", data
+end
+
+escalation "esc_terminate_instance" do
+  automatic false
+  label "Terminate Instance"
+  description "Terminate or delete the Instance"
+  run "terminate_instance", data
+end
+
+escalation "esc_update_schedule" do
+  automatic false
+  label "Update Schedule"
+  description "Update the existing schedule Tag"
+  parameter "param_schedule" do
+    type "string"
+    label "New Schedule"
+    description "Provide Schedule Value"
+  end
+  run "update_schedule", data, $param_schedule
+end
+
+escalation "esc_delete_schedule" do
+  automatic false
+  label "Delete Schedule"
+  description "Delete Schedule Tag"
+  run "delete_schedule", data
+end
+
+###############################################################################
+# Policy
+###############################################################################
+
+policy "policy_schedule_instance" do
+  validate $ds_filter_instances do
+    summary_template "Azure Schedule Instance list"
+    hash_include "next_start", "next_stop"
+    escalate $esc_email
+    escalate $esc_schedule_instance
+    escalate $esc_terminate_instance
+    escalate $esc_update_schedule
+    escalate $esc_delete_schedule
+    check eq(size(data),0)
+    export do
+      resource_level true
+      field "id" do
+        label "Instance ID"
+      end
+      field "name" do
+        label "Instance Name"
+      end
+      field "rg" do
+        label "Resource Group"
+      end
+      field "location" do
+        label "Location"
+      end  
+      field "all_tags" do
+        label "Tags"
+      end
+      field "state" do
+        label "State"
+      end
+      field "schedule" do
+        label "Schedule"
+      end
+      field "subscriptionId" do
+        label "SubscriptionId"
+      end
+      field "subscriptionName" do
```

Comment:
subscription name should be the first column and sorted.

Reply:
Do we need to sort the subscription, since now we can sort in UI ?

Reply:
Why wouldn't we want to have the subscription sorted in the default view?

Reply:
Yes we can do that, I will update the template.